docs/README.md

```diff
@@ -17,6 +17,7 @@
   * [Haskell (haskell)](./docs/Options.md#haskell-haskell)
   * [Pyenv (pyenv)](./docs/Options.md#pyenv-pyenv)
   * [Go (golang)](./docs/Options.md#go-golang)
+  * [Kubectl (kubectl)](./docs/Options.md#kubectl)
```

Comment:
Like in the docs of spaceship, I would change the text here to `Kubectl context (kubecontext)`. You will need to then change the anchor to `#kubectl-context-kubecontext`. 🔗